MobileWallet/Screens/Send/AddRecipient/ContactsTableViewController.swift

```diff
@@ -48,8 +48,30 @@ protocol ContactsTableDelegate {
 class ContactsTableViewController: UITableViewController {
     var actionDelegate: ContactsTableDelegate?
 
+    var filter: String = "" {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+
     private var recentContactList: [Contact] = []
     private var contactList: [Contact] = []
+
+    private var filteredRecentContactList: [Contact] {
```

Comment:
making this a computed property might not be a good idea because for every cell, every cellforrowatindexpath call this will be called and calculated again

Reply:
Have updated this to set the filtered lists each time the `filter` string is set